core/src/scrim/scrim.mod.resolver.ts

```diff
@@ -228,7 +228,14 @@ export class ScrimModuleResolver {
      *
      */
 
-    @Subscription(() => ScrimEvent)
+    @Subscription(() => ScrimEvent, {
+        async filter(this: ScrimModuleResolver, payload: {followCurrentScrim: ScrimEvent;}, variables, context: {req: {user: UserPayload;};}) {
+            const {userId, currentOrganizationId} = context.req.user;
+            if (!currentOrganizationId) return false;
+            const scrim = await this.scrimService.getScrimByPlayer(userId);
+            return scrim?.submissionId === payload.followCurrentScrim.scrim.submissionId;
+        },
+    })
```

Comment:
Is the submission coming from this resolver? I think there's a separate one for submissions